modules/manage/pages/iceberg/iceberg-topics-aws-glue.adoc

```diff
@@ -0,0 +1,191 @@
+= Query Iceberg Topics using AWS Glue
+:description: Add Redpanda topics as Iceberg tables that you can query from AWS Glue Data Catalog.
+:page-categories: Iceberg, Tiered Storage, Management, High Availability, Data Replication, Integration
+
+[NOTE]
+====
+include::shared:partial$enterprise-license.adoc[]
+====
+
+// tag::single-source[]
+
+This guide walks you through querying Redpanda topics as Iceberg tables stored in AWS S3, using a catalog integration with https://docs.aws.amazon.com/glue/latest/dg/components-overview.html#data-catalog-intro[AWS Glue^]. For general information about Iceberg catalog integrations in Redpanda, see xref:manage:iceberg/use-iceberg-catalogs.adoc[].
+
+== Prerequisites
+
+* Redpanda version 25.1.7 or later.
+ifndef::env-cloud[]
+* xref:manage:tiered-storage.adoc#configure-object-storage[Object storage configured] for your cluster and xref:manage:tiered-storage.adoc#enable-tiered-storage[Tiered Storage enabled] for the topics for which you want to generate Iceberg tables.
++
+You also use the S3 bucket URI to set the base location for Glue Data Catalog.
+endif::[]
+
+== Limitations
+
+=== Nested partition spec support
+
+Glue does not support partitioning on nested fields. If Redpanda detects that
+the default partitioning `(hour(redpanda.timestamp))` is in use, it will instead apply an empty partition spec `()`, which means the table will not be partitioned.
+
+If you want to use partitioning, you must specify a custom partition specification using your own partition columns (columns that are not nested).
+
+== Authorize access to AWS Glue
+
+You need to allow Redpanda access to Glue services in your AWS account. You can use the same access credentials that you configured for S3 (IAM role, access keys, and KMS key), as long as you have also added read and write access to Glue Data Catalog.
+
+For example, you could create an IAM policy that manages access to Glue, and attach it to the IAM role that Redpanda also uses to access S3. It is recommended to add all Glue actions in the policy (`"glue:*"`), on the following resources:
```

Comment:
@kbatuigas this explanation is good, but I wonder if there are AWS Glue docs  we should link to that would be helpful here as well